samples/java-spring-eventsourced-customer-registry-subscriber/README.md

```diff
@@ -13,38 +13,46 @@ Use Maven to build your project:
 ```shell
 mvn compile
 ```
+
 ## Running Locally
 
 First start the `java-spring-eventsourced-customer-registry` service and proxy. It will run with the default service and proxy ports (`8080` and `9000`).
 
-To start the proxy, run the following command from this directory:
+To start the application locally, the `kalix-maven-plugin` is used. Use the following command:
 
 ```shell
-docker-compose up
+mvn kalix:runAll
```

Comment:
I'm getting 
```
[ERROR] Could not find goal 'runAll' in plugin io.kalix:kalix-maven-plugin:1.2.0 among available goals deploy, generate, help -> [Help 1]
org.apache.maven.plugin.MojoNotFoundException: Could not find goal 'runAll' in plugin io.kalix:kalix-maven-plugin:1.2.0 among available goals deploy, generate, help
```
Although I can see the `goal` in 1.2.0 already

Reply:
You have to build the SDK locally. 'runAll' is not released yet.

Reply:
If you want to run it locally, you need to first:

`./publishLocally.sh`

Then you need to publish the `customer-register` docker image to your local docker registry. Watch out with that. If you are using Apple M1/2 processors, it will be very slow and the tests will probably not even pass. Let me know if that's the case and I show you the trick to build a arm64 image for `customer-register`. 
I will later send a PR to improve the docker build for those services.

And then you need to update the subscriber pom.xml to use the new snapshot. After that, you can run: `mvn -Pit verify` in the subscriber project.

Reply:
roger that

Reply:
Works like a charm. This improves DX a lot. 
Regarding the `customer-register` sample. How do you find to update the README to use this feature as well in there so that in both places one uses `mvn kalix:runAll`? So far `java-spring-eventsourced-customer-registry` stills points to the `docker compose up`.

Reply:
We will have a follow up PR that will update all the samples and update the docs to explain `mvn kalix:runAll`. This PR is more about testing for local service discovery in dev-mode. 

The `kalix:runAll` and local service discovery are features added in previous PRs. And we were missing the testing part. 

Next step will be to document `kalix:runAll` and how to do multi-service testing.